docs/resources/aws_vpn_connections.md

```diff
@@ -0,0 +1,67 @@
+---
+title: About the aws_vpn_connections Resource
+platform: aws
+---
+
+# aws\_vpn\_connections
+
+Use the `aws_vpn_connections` InSpec audit resource to test properties of all AWS VPN Connections.
+
+## Syntax
+
+Ensure that one or more VPN Connections exist.
+
+    describe aws_vpn_connections do
+      it { should exist }
+    end
+
+## Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on VPN Connections](https://docs.aws.amazon.com/vpc/latest/userguide/vpn-connections.html).
+
+## Properties
+
+|Property           | Description|
+| ---               | --- |
+|vpn\_connection\_id    | This property provides all the IDs of the VPN Connections |
+|vpn\_gateway\_id       | This property provides all the IDs of the virtual private gateways associated with the VPN Connections |
+|tunnel\_options        | This property provides the tunnel options of the VPN Connections. |
+|state                  | This property provides the current state of the VPN Connections |
+
+## Examples
+
+##### Ensure that VPN connections are available.
+    describe aws_vpn_connections do
+      its('state') { should eq 'available' }
```

Comment:
The `state` property will be an array, that's why matching it to a string will fail in any case.
If you want to test that all VPN connections' states are `available` this test can be updated as:
`its('state.uniq') { should eq ['available'] }`.